starter-repos/app-template/.gitlab-ci.yml

```diff
@@ -0,0 +1,21 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+include:
+- '/ci/go-test.yaml'
```

Comment:
Shall we reuse the shared-ci-cd, like other templates?

Reply:
The shared-ci-cd doesn't work for this example.
1. the golang version isn't the same
2. need to set the project: 'platform-admins/shared-ci-cd'

Reply:
We could update the shared-ci-cd to use golang v 1.13, right? 

> need to set the project: 'platform-admins/shared-ci-cd'

What's blocking us from setting the project in this template?

Reply:
Yes, we can update the shared-ci-cd on this branch. I forgot about that.

The `project 'platform-admins/shared-ci-cd'` is another private repo on the same gitlab instance. For example, if the application is put in the gitlab repo `gitlab.${HOST}.com/myteam/app.git`. The shared-ci-cd should be in the repo `gitlab.${HOST}.com/platform-admins/shared-ci-cd.git` Users need to set this up before they initialize the application. I'm not sure if we list it as a prerequiste.

Reply:
IIUC the `./anthos-platform-cli add app` command copies the `platform-admins/shared-ci-cd` folder in this repo to the users' GitLab instance.

Reply:
Cool! Thank I'll change it to reuse the shared-ci-cd.

Reply:
Correction: copying the project is done by Terraform, not anthos-platform-cli.

Reply:
@janetkuo I don't see part mentioned in the demo doc. Should we add it as a prerequisite?